assets/xml/scenes/dungeons/jyasinzou.xml

```diff
@@ -6,6 +6,10 @@
         <Scene Name="jyasinzou_scene" Offset="0x0"/>
         <Scene Name="jyasinzou_scene_unused1" Offset="0x17220"/>
         <Scene Name="jyasinzou_scene_unused2" Offset="0x17710"/>
+
+        <!-- Note: these cutscenes are the same as `gKokiriForestKokiriEmeraldPart7Cs` -->
+        <Cutscene Name="gSpiritTempleUnused1Cs" Offset="0x16860"/>
+        <Cutscene Name="gSpiritTempleUnused2Cs" Offset="0x16D40"/>
```

Comment:
They arent exactly the same, they have very slight differences. I think better wording could be "These are unused versions of `gKokiriForestKokiriEmeraldPart7Cs`"